packages/blue-migration-sdk-viem/README.md

```diff
@@ -0,0 +1,34 @@
+# @morpho-org/blue-migration-sdk-viem
+
+<a href="https://www.npmjs.com/package/@morpho-org/blue-migration-sdk-viem">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/npm/v/@morpho-org/blue-migration-sdk-viem?colorA=21262d&colorB=21262d&style=flat">
+        <img src="https://img.shields.io/npm/v/@morpho-org/blue-migration-sdk-viem?colorA=f6f8fa&colorB=f6f8fa&style=flat" alt="Version">
+    </picture>
+</a>
+<a href="https://github.com/morpho-org/blue-migration-sdk-viem/blob/main/LICENSE">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/npm/l/@morpho-org/blue-migration-sdk-viem?colorA=21262d&colorB=21262d&style=flat">
+        <img src="https://img.shields.io/npm/l/@morpho-org/blue-migration-sdk-viem?colorA=f6f8fa&colorB=f6f8fa&style=flat" alt="MIT License">
+    </picture>
+</a>
+<a href="https://www.npmjs.com/package/@morpho-org/blue-migration-sdk-viem">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset="https://img.shields.io/npm/dm/@morpho-org/blue-migration-sdk-viem?colorA=21262d&colorB=21262d&style=flat">
+        <img src="https://img.shields.io/npm/dm/@morpho-org/blue-migration-sdk-viem?colorA=f6f8fa&colorB=f6f8fa&style=flat" alt="Downloads per month">
+    </picture>
+</a>
+<br />
+<br />
+
+Viem-based extension of `@morpho-org/simulation-sdk` that exports utilities to build migration bundles to migrate lending positions (on aave, compound, morpho-aaveV3-optimizer, ...) to the morpho protocol.
+
+## Installation
+
+```bash
+npm install @morpho-org/blue-migration-sdk-viem
```

Comment:
```suggestion
npm install @morpho-org/migration-sdk-viem
```

Can we call it `migration-sdk-viem`? To make it shorter and consistent with other sdks

Reply:
In my opinion the other packages are the wrongly named ones :/ In the context of these sdks it is obvious that it's all related to `blue` but as a consumer interacting with several protocols, we lose this information

Will be even worse when we'll start to work on morpho v2